nix/pkgs/core.nix

```diff
@@ -17,14 +17,14 @@ rec {
     export CARGO_BUILD_JOBS=4
 
     # ensure plain build works
-    cargo build --no-default-features --manifest-path=crates/holochain/Cargo.toml
+    # cargo build --no-default-features --manifest-path=crates/holochain/Cargo.toml
```

Comment:
I'm fine removing this as it doesn't fully simulate the Nix builds anyway. Please remove this line and its comment altogether.